test/Azure.Functions.Cli.Tests/ActionsTests/PackActionTests.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Actions.LocalActions;
+using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Interfaces;
+using Moq;
+using Xunit;
+
+namespace Azure.Functions.Cli.Tests.ActionsTests
+{
+    public class PackActionTests
```

Comment:
We have a UnitTest project that has a ZipHelpers test, can we expand that test to unit test the zip functionality that might be missing?